opencti-platform/opencti-graphql/src/modules/case/feedback/feedback-resolvers.ts

```diff
@@ -0,0 +1,50 @@
+import type { Resolvers } from '../../../generated/graphql';
+import { buildRefRelationKey } from '../../../schema/general';
+import { RELATION_CREATED_BY, RELATION_OBJECT_ASSIGNEE, RELATION_OBJECT_LABEL, RELATION_OBJECT_MARKING } from '../../../schema/stixMetaRelationship';
+import { stixDomainObjectAddRelation, stixDomainObjectCleanContext, stixDomainObjectDelete, stixDomainObjectDeleteRelation, stixDomainObjectEditContext, stixDomainObjectEditField } from '../../../domain/stixDomainObject';
+import { addFeedback, feedbackContainsStixObjectOrStixRelationship, findAll, findById } from './feedback-domain';
+
+const feedbackResolvers: Resolvers = {
+  Query: {
+    feedback: (_, { id }, context) => findById(context, context.user, id),
+    feedbacks: (_, args, context) => findAll(context, context.user, args),
+    feedbackContainsStixObjectOrStixRelationship: (_, args, context) => {
```

Comment:
I don't find where this method is used

Reply:
Same as the case incident. It was a query made in client-python